assets/scss/core/_variables.scss

```diff
@@ -25,26 +26,33 @@ $dp-font-color-dark: $dp-font-color !default;
 $dp-font-color-white: #fff !default;
 
 
-$dp-font-size-xxl: 60px;
-$dp-headers-h1-font-size: 50px;
-$dp-headers-h2-font-size: 40px;
-$dp-headers-h3-font-size: 30px;
-$dp-headers-h4-font-size: 22px;
-$dp-form-font-size: 20px;
-$dp-headers-h5-font-size: 18px;
-$dp-headers-h6-font-size: 13px;
-$dp-font-size-small: 13px;
-$dp-font-size-xsmall: 11px;
-$dp-font-size-form: 14px;
-$dp-font-button-big: 15px;
-$dp-font-button-medium: 13px;
-$dp-font-button-small: 11px;
-$dp-font-size-15: 15px;
-$dp-font-size-24: 24px;
-$dp-font-small-lineheight: 18px;
+////
+/// Font Size settings
+///
+
+$dp-font-size--lv0: 0;
```

Comment:
Does this variable make sense?

Reply:
En algunos casos necesitamos ocultar el contenido, actualmente pasa con el logo en signup, el navegador lo lee pero no tiene tamaño el contenido y se coloca la imagen por background

Reply:
Thanks for the explanation, you are the expert.